lib/urbanopt/geojson/schema/region_properties.json

```diff
@@ -6,46 +6,28 @@
   "type": "object",
   "properties": {
     "id": {
-      "FIXME": "may be there, type not known (string? number?)",
-      "description": "Unique id used to refer to this feature within this dataset, generated by database.",
+      "description": "Unique id used to refer to this feature within this dataset.",
       "type": "string"
     },
     "project_id": {
-      "description": "Project which this feature belongs to, managed by the database.",
-      "type": "string"
-    },
-    "created_at": {
-      "description": "Generated by database, YYYY-MM-DDTHH:MM:SS.SSSZ",
-      "type": "string"
-    },
-    "updated_at": {
-      "description": "Generated by database, YYYY-MM-DDTHH:MM:SS.SSSZ",
-      "type": "string"
-    },
-    "geojson_id": {
-      "description": "Id of the geojson for this feature, managed by the database.",
+      "description": "Project which this feature belongs to.",
       "type": "string"
     },
     "type": {
       "description": "Type of feature.",
       "type": "string",
       "enum": [
-        "Building",
-        "Taxlot",
-        "Region",
-        "District System", 
-        "Connector"
+        "Region"
       ]
     },
     "region_type": {
       "type": "string",
       "enum": [
-        "State",
-        "County",
-        "CensusTract",
-        "CensusBlockGroup",
-        "CensusBlock",
-        "Custom"
+        "Taxlot",
+        "Park",
+        "Road",
+        "Lake",
+        "Obstacle"
```

Comment:
@amyeallen1 ... will this work to denote your constraints?

Reply:
@nllong @macumber Thanks for checking. Yes, those descriptions seem fine. Would one be able to use the coordinates associated with a "region" later on as a constraint for routing infrastructure?

Reply:
you'd have to implement checks that things don't intersect, i believe we can do that in the ui